package.json

```diff
@@ -1,21 +1,34 @@
 {
   "name": "babel-plugin-istanbul",
-  "version": "1.0.0-alpha.1",
+  "version": "1.0.0",
```

Comment:
@rmg I've been using [standard-version](https://www.npmjs.com/package/standard-version) for bumping releases/creating change-logs lately. main reason I changed this before landing was, I'm not quite sure how the tool would deal with pre-release tags ... I should probably dig into this.